static/haven/example_3_person_invalid/server_request.json

```diff
@@ -1,5 +1,7 @@
+// these JSON became no longer invalid when requirements for "physicalhousehold" changed
+// so I have changed them to being invalidated on their values for "taxhousehold"
```

Comment:
The problem here is that if the client request is invalid because the tax household relationships don't match the tax household info in Person, our server will immediately send an error back to the client without calling HAVEN. So there should be no server request or server response file! @patharveybtr